projects/angular/storyteller/src/app/routes/user-info/user-info.component.scss

```diff
@@ -0,0 +1,46 @@
+.user-info-page {
```

Comment:
I deleted 'feeds.scss`, since there's no need in it.
If you'd like to apply a style to whole app, you can create a class in global styles. It's not necessary to import feeds in a each one component.

Reply:
But don't get carried away, each component usually is unique element. It shouldn't have a lot of same parts with another component. If there's something similiar, it's probably can be shared.